integration_tests/model_visualization_test.py

```diff
@@ -329,7 +358,6 @@ def call(self, xs):
 
 
 if __name__ == "__main__":
-    plot_sequential_model()
```

Comment:
Why remove this one?

Reply:
The reason why I removed that line is I want to change this file running format as pytest. So, I added assert statement to check file exists and I have a plan to check model structure also by using model_to_dot method and assert statement. Finally, I want to remove all main statement from this file, and run this by pytest comand.
However, if you think these main statement should be exist, It is also ok not to remove that.

Reply:
Yes, please leave it

Reply:
Yes, I did it. Thanks.